tests/functional/test_new_resolver_target.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from pip._internal.cli.status_codes import ERROR, SUCCESS
+from tests.lib.path import Path
+from tests.lib.wheel import make_wheel
+
+
+@pytest.fixture()
```

Comment:
nit: we can drop the `()` if there are no arguments

Reply:
I like to keep them since it keeps the call style consistent no matter there’s arguments or not 🙂